lcls_tools/common/devices/reader.py

```diff
@@ -8,8 +8,8 @@
 from lcls_tools.common.devices.area import Area
 from lcls_tools.common.devices.beampath import Beampath
 
-
-DEFAULT_YAML_LOCATION = "./lcls_tools/common/devices/yaml/"
+DEFAULT_YAML_LOCATION = os.path.join(os.path.dirname(__file__), "yaml/")
```

Comment:
This is a good change, allows people to set `lcls_tools.common.devices.reader.DEFAULT_YAML_LOCATION` to a different location in their scripts if that's needed too!